### ISSUE TITLE
Sample UI

### ISSUE THREAD

**Issue by jpietersma** [Software-Engineering-CSUSM/Test-Taker#13]
I'm not sure if this is the right place for this, but I was able to put together a sample UI.  It took me a little while to figure it out, but I was able to get something together.  I also uploaded the code in my branch "Jeremy".  However, I haven't uploaded the image.

<img width="960" alt="ui pic" src="https://cloud.githubusercontent.com/assets/21963401/18860487/9829acac-8432-11e6-9249-0047c1421e4c.png">

**Comment by jstngoulet** [comment]
Coming along great! Were you able to see my sample code?

**Comment by jpietersma** [comment]
Yeah, I saw it thanks.  Also just uploaded the image in my branch.
